pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -722,3 +723,61 @@ def partition(low, high, pick_pivot_element):
         array._modify(force=True)
 
     return array
+
+def longest_common_subsequence(seq1, seq2) -> tuple:
+    """
+    Implements Longest Common Subsequence
+
+    Parameters
+    ========
+
+    seq1: string or list
+    seq2: string or list
+
+    Returns
+    =======
+
+    output: tuple
+    (Length of LCS, Common Sequence)
+    Common Sequence will be of the same data type is the seq1.
```

Comment:
Common Sequence will be of the same data type as of seq1.

Reply:
> As per docstring , input of function should either be a string or a list. So we should check the type of input before proceeding .

Please elaborate the issue I'm not getting it

Reply:
> Common Sequence will be of the same data type as of seq1.

This is just improvement in English of docstring.

Reply:
Yep done too